### PR TITLE
Team F: Multi robot Navigation stack design

### PR DESCRIPTION
**Pull request by indraneelpatil**
1. We compared two competing approaches for centralised multi robot navigation and found the control based approach to work better
2. We discussed design of each and pros and cons of both the approaches
3. We hope this discussion provides a good foundation for the design of multi robot navigation stacks of MRSD in the future!

### CODE REVIEW

File: wiki/planning/multi-robot-planning.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+date: {}
```
Comment (smbryan) [inline wiki/planning/multi-robot-planning.md:2]: Date?

CC: @nevalsar

**Review by smbryan — CHANGES_REQUESTED** [review]
One item to address, then this should be ready to merge. Great submission.

File: wiki/planning/multi-robot-planning.md (hunk 1 above)
Comment (indraneelpatil) [reply wiki/planning/multi-robot-planning.md]: Done! @smbryan @nevalsar Thanks!

**Comment by nevalsar** [comment]
That should fix it. Merging.

**Pull request merged**